.github/workflows/post-deploy.yml

```diff
@@ -0,0 +1,95 @@
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  workflow_call:
+    inputs:
+      assets_required:
+        required: false
+        default: false
+        type: boolean
+      postgres_unit_testing:
+        required: false
+        default: false
+        type: boolean
+      db_name:
+        required: false
+        default: "postgres_db"
+        type: string
+      run_performance_tests:
+        required: false
+        default: false
+        type: boolean
+      run_e2e_tests:
+        required: false
+        default: false
+        type: boolean
+
+    # not required when not deploying to cf
```

Comment:
This workflow shouldn't be touching cloudfoundry anymore, so safe to remove this